docs/getting-started/installation.md

```diff
@@ -50,19 +50,21 @@ Follow the instructions in the [React documentation to set up a new project with
 
 Once your project is set up and React Router is installed as a dependency, open the `src/index.js` in your text editor. Import `BrowserRouter` from `react-router-dom` near the top of your file and wrap your app in a `<BrowserRouter>`:
 
-```js [3, 9-11]
+```js [3, 11-13]
 import * as React from "react";
-import * as ReactDOM from "react-dom";
+import ReactDOM from "react-dom/client";
 import { BrowserRouter } from "react-router-dom";
 import "./index.css";
 import App from "./App";
-import * as serviceWorker from "./serviceWorker";
+import reportWebVitals from "./reportWebVitals";
```

Comment:
I removed 
```js
import * as serviceWorker from "./serviceWorker";
```
and add this one
```js
import reportWebVitals from "./reportWebVitals";
```
 because I think most people will use create-react-app without PWA template.